templates/taxi/car_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Car</h1>
+
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
+{% endblock %}

```

Comment:
Make sure you've added a blank line at the end of the file as per the code style checklist.